core/src/franchise/player/player.controller.ts

```diff
@@ -154,4 +166,77 @@ export class PlayerController {
 
         return "SUCCESS";
     }
+
+    @MessagePattern(CoreEndpoint.GetPlayerByPlatformId)
+    async getPlayerByPlatformId(@Payload() payload: unknown): Promise<CoreOutput<CoreEndpoint.GetPlayerByPlatformId>> {
+        const data = CoreSchemas[CoreEndpoint.GetPlayerByPlatformId].input.parse(payload);
+
+        const game = await this.gameService.getGameById(data.gameId);
+        const platform = await this.platformService.getPlatformByCode(data.platform);
+        const player = await this.playerService.getPlayerByGameAndPlatform(game.id, platform.id, data.platformId).catch(() => null);
+
+        if (!player) return {
+            success: false,
+            request: data,
+        };
+
+        const mlePlayer = await this.playerService.getMlePlayerBySprocketPlayer(player.id);
+
+        return {
+            success: true,
+            data: {
+                id: player.id,
+                userId: player.member.user.id,
+                skillGroupId: player.skillGroupId,
+                franchise: {
+                    name: mlePlayer.teamName,
+                },
+            },
+            request: data,
+        };
+    }
+
+    @MessagePattern(CoreEndpoint.GetPlayersByPlatformIds)
+    async getPlayersByPlatformIds(@Payload() payload: unknown): Promise<CoreOutput<CoreEndpoint.GetPlayersByPlatformIds>> {
+        const data = CoreSchemas[CoreEndpoint.GetPlayersByPlatformIds].input.parse(payload);
+
+        const allResults = await Promise.allSettled(data.map(async p => {
+            const game = await this.gameService.getGameById(p.gameId);
+            const platform = await this.platformService.getPlatformByCode(p.platform);
+            const player = await this.playerService.getPlayerByGameAndPlatform(game.id, platform.id, p.platformId).catch(() => null);
+    
+            if (!player) return {
+                success: false,
+                error: `Failed to find player by account platform=${p.platform} platformId=${p.platformId}`,
+                request: p,
+            };
+    
+            const mlePlayer = await this.playerService.getMlePlayerBySprocketPlayer(player.id);
+    
+            return {
+                success: true,
+                data: {
+                    id: player.id,
+                    userId: player.member.user.id,
+                    skillGroupId: player.skillGroupId,
+                    franchise: {
+                        name: mlePlayer.teamName,
+                    },
+                },
+                request: p,
+            };
```

Comment:
Duplicated code from `getPlayerByPlatformId`, this could be a function in `playerService`